### DISCUSSION TITLE
[Feature Request] Repository Grouping

### DISCUSSION THREAD

**Discussion by nbrugger-tgm** [community/community#7354] [Other Feature Feedback, Questions, & Ideas]
Hello,

My proposal is that there is an option to group "profile/organisation" level repositories together in groups.

### Use Case

I have a few different types of repositories like:
- Java libraries
- Project templates
- Experiments
- API Clients

If users want to skim over my repos they are most likely just interested in projects of a certain kind, for example my java-libraries.

**Comment by cypher7682** [comment]
Basically copy paste what gitlab are doing, because that is *fantastic*

**Comment by NovaNix** [comment]
Organizing my repositories in folders would be a dream. Right now it's just an unstructured blob. Please add this.

**Comment by Suvraneel** [comment]
This'll be a very cool feature ngl

**Comment by MarcoValliIMA** [comment]
Similar discussion can be found here [ISSUE 4174](https://github.com/github/feedback/discussions/4174)

**Reply by nbrugger-tgm** [reply]
Oh sorry, as I see my post is the duplicate of a duplicate of a duplicate and the original post about this feature is.... 7.... Years old...

Yeah - time to move on

Sorry for the rant
